Makefile

```
@@ -1,3 +1,7 @@
+DEFAULT_GOARCH := $(shell go env GOARCH)
+BUILD_HASH = $(shell git rev-parse HEAD)
+LDFLAGS += -X "github.com/mattermost/rtcd/service.buildHash=$(BUILD_HASH)"
```

I think we can simplify to:
```suggestion
LDFLAGS += -X "service.buildHash=$(BUILD_HASH)"
```

---

Interestingly I wasn't able to make it work that way and went with the full module version. I think the short form works if you are targeting the `main` package but not otherwise.